brd-android/fabriik-common/src/main/java/com/fabriik/common/data/model/Profile.kt

```diff
@@ -54,6 +54,13 @@ fun Profile?.availableDailyLimit(): BigDecimal = this?.exchangeLimits?.available
 
 fun Profile?.availableLifetimeLimit(): BigDecimal = this?.exchangeLimits?.availableLifetime() ?: BigDecimal.ZERO
 
+fun Profile?.kyc2ExchangeLimit(): BigDecimal? {
```

Comment:
All this methods consider the possibility of the this object to be null. Why is that required?

Reply:
Profile can be null if user is not registered. I added extension functions to nullable Profile object to avoid setting default value everytime these methods are called.